components/Header/Header.tsx

```diff
@@ -292,7 +292,7 @@ const NotificationIconWrapper = styled.div`
   }
 `;
 
-const Dot = styled.div`
+const Dot = styled.span`
```

Comment:
these divs were resulting in invalid html because div cannot be a child of p. Changed to span to fix, functionality is identical